de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -744,7 +777,7 @@ private boolean terminateNegotiations() {
 
         while (System.currentTimeMillis() - startTime < NEGOTIATION_TIMEOUT) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reduce the total number of break and continue statements in this loop to use at most one. ([squid:S135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=272&issues=AWbfy2AyLpqqSUlcop4E&open=AWbfy2AyLpqqSUlcop4E)

[ISSUE_KEY]: AWbfy2AyLpqqSUlcop4E